### ISSUE TITLE
Crashreport Android 5.0

### ISSUE THREAD

**Issue by milovanderlinden** [CodeForEindhoven/open311-android#41]
```java
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.open311.android/org.open311.android.MainActivity}: android.content.res.Resources$NotFoundException: File res/drawable-anydpi-v21/ic_add_alert_white_24dp.xml from drawable resource ID #0x7f02005e
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2689)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2754)
	at android.app.ActivityThread.access$900(ActivityThread.java:177)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1448)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:145)
	at android.app.ActivityThread.main(ActivityThread.java:5938)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1400)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1195)
Caused by: android.content.res.Resources$NotFoundException: File res/drawable-anydpi-v21/ic_add_alert_white_24dp.xml from drawable resource ID #0x7f02005e
	at android.content.res.Resources.loadDrawableForCookie(Resources.java:3735)
	at android.content.res.Resources.loadDrawable(Resources.java:3603)
	at android.content.res.Resources.getDrawable(Resources.java:1852)
	at android.content.Context.getDrawable(Context.java:410)
	at android.support.v4.content.ContextCompatApi21.getDrawable(ContextCompatApi21.java:26)
	at android.support.v4.content.ContextCompat.getDrawable(ContextCompat.java:344)
	at org.open311.android.adapters.ViewPagerAdapter.getPageTitle(ViewPagerAdapter.java:49)
	at android.support.design.widget.TabLayout.populateFromPagerAdapter(TabLayout.java:903)
	at android.support.design.widget.TabLayout.setPagerAdapter(TabLayout.java:894)
	at android.support.design.widget.TabLayout.setupWithViewPager(TabLayout.java:807)
	at android.support.design.widget.TabLayout.setupWithViewPager(TabLayout.java:768)
	at android.support.design.widget.TabLayout.setupWithViewPager(TabLayout.java:746)
	at org.open311.android.MainActivity.onCreate(MainActivity.java:94)
	at android.app.Activity.performCreate(Activity.java:6288)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1119)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2642)
	... 10 more
Caused by: java.lang.NumberFormatException: Invalid float: "1.1.89"
	at java.lang.StringToReal.invalidReal(StringToReal.java:63)
	at java.lang.StringToReal.initialParse(StringToReal.java:160)
	at java.lang.StringToReal.parseFloat(StringToReal.java:323)
	at java.lang.Float.parseFloat(Float.java:306)
	at android.util.PathParser.getFloats(PathParser.java:178)
	at android.util.PathParser.createNodesFromPathData(PathParser.java:59)
	at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1439)
	at android.graphics.drawable.VectorDrawable$VFullPath.inflate(VectorDrawable.java:1421)
	at android.graphics.drawable.VectorDrawable.inflateInternal(VectorDrawable.java:542)
	at android.graphics.drawable.VectorDrawable.inflate(VectorDrawable.java:458)
	at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1150)
	at android.graphics.drawable.Drawable.createFromXml(Drawable.java:1063)
	at android.content.res.Resources.loadDrawableForCookie(Resources.java:3719)
	... 25 more
```

**Comment by milovanderlinden** [comment]
Related to a known issue: https://code.google.com/p/android/issues/detail?id=202670

**Comment by milovanderlinden** [comment]
Fixed in the 1.4-Alpha branch
